### PR TITLE
Add event chapter banner

### PR DESCRIPTION
**Pull request by BasVoorhoede**
Preview to event chapter page: https://deploy-preview-196--wwa.netlify.app/en/events/shenzhen-design-week/climate-change-and-migration
Preview to event chapter page without image: https://deploy-preview-196--wwa.netlify.app/en/events/shenzhen-design-week/impacts-on-coastal-safety

### CODE REVIEW

File: components/narrative-header/NarrativeHeaderEvent.vue [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+      class="narrative-header-event__navigation"
+    >
+      <nuxt-link
+        :to="`/${this.$route.params.language}/events/${this.$route.params.event}`"
+        class="narrative-header-event__back-button"
+      >
+        Back
+      </nuxt-link>
+
+      <select
```
Comment (jbmoelker) [inline components/narrative-header/NarrativeHeaderEvent.vue:28]: Should we use the same component as in the regular NarrativeHeader?
Screen Shot 2020-07-08 at 09 59 39

File: components/narrative-header/NarrativeHeaderEvent.vue [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    background-color: var(--blue-secondary);
+    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 154 84' xmlns='http://www.w3.org/2000/svg'%3E%3Cg stroke='%23FFF' fill='none' fill-rule='evenodd' opacity='.4'%3E%3Cpath d='M-201.3 224c36.4-42.2 60-66.2 182.7-58 75.1 4.9 72.1-74.2 187.4-91.6'/%3E%3Cpath d='M-202.3 217.2c36.5-42 60-66 182.8-58 75 5 72-74 187.4-91.5'/%3E%3Cpath d='M-203.2 210.5c36.5-42 60-66.1 182.7-58 75.1 5 72.1-74.1 187.4-91.5'/%3E%3Cpath d='M-204.1 203.8c36.4-42.1 60-66.1 182.7-58 75.1 5 72.1-74.1 187.4-91.5'/%3E%3Cpath d='M-205 197c36.4-42 60-66 182.7-58 75 5 72-74 187.4-91.4'/%3E%3Cpath d='M-206 190.4c36.5-42.1 60-66.1 182.7-58 75.2 5 72.1-74.1 187.5-91.5'/%3E%3Cpath d='M-207 183.6c36.5-42 60-66 182.8-58 75.1 5 72.1-74 187.4-91.4'/%3E%3Cpath d='M-207.9 177c36.5-42.2 60-66.2 182.8-58 75 4.9 72-74.2 187.4-91.6'/%3E%3Cpath d='M-208.8 170.2c36.5-42 60-66 182.7-58 75.2 5 72.1-74 187.5-91.5'/%3E%3Cpath d='M-209.7 163.5c36.4-42 60-66.1 182.7-58 75.1 5 72.1-74.1 187.4-91.5'/%3E%3Cpath d='M-210.7 156.8c36.5-42.1 60-66.1 182.8-58 75 5 72-74.1 187.4-91.5'/%3E%3Cpath d='M-211.6 150c36.5-42 60-66 182.7-58C46.3 97 43.3 18 158.6.7'/%3E%3C/g%3E%3C/svg%3E");
```
Comment (jbmoelker) [inline components/narrative-header/NarrativeHeaderEvent.vue:139]: I think you can solve this the same as you're doing above with `background-image: url(/assets/event-wave.svg)`. Nuxt will inline the svg for you when it's small enough. This keeps the CSS easier to understand.

Same with inlined SVG's below?

File: pages/_language/events/_event/_chapter.vue [hunk 3]
```diff
@@ -60,11 +87,12 @@
         params,
       };
     },
+    methods: {
+      smoothScroll (slug) {
+        const element = this.$refs[slug][0];
```
Comment (jbmoelker) [inline pages/_language/events/_event/_chapter.vue:92]: easier to use `scrollIntoView`? https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Comment (BasVoorhoede) [reply pages/_language/events/_event/_chapter.vue:92]: This is not supported in Safari using the 'smooth' behavior option.

**Pull request merged**